src/ansys/dpf/core/operators/result/equivalent_mass.py

```diff
@@ -54,9 +54,6 @@ class equivalent_mass(Operator):
     data_sources : DataSources
         Result file path container, used if no
         streams are set
-    bool_rotate_to_global : bool, optional
```

Comment:
Hi @oparreno, do you know why this pin was removed? It looks to me like this is not retro-compatible.

Reply:
@PProfizi, this is because equivalent_mass can't be rotated to global as it was done, that was a bug, thus we disabled it and deleted the pin.